internal/runners/run/run.go

```diff
@@ -145,11 +158,20 @@ func configCachePath() string {
 	return config.CachePath()
 }
 
-func pathProvidesExec(filterByPath, exec, path string) bool {
+func pathProvidesLang(path string, language language.Language) bool {
 	paths := splitPath(path)
-	if filterByPath != "" {
-		paths = filterPrefixed(filterByPath, paths)
```

Comment:
Wouldn't you still want this?

Reply:
No not with the updates, though I've since updated the caller of this function and hopefully it's more clear now